src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -238,4 +239,23 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
             return null;
         }
     }
+    public class BooleanToIntConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if(value is bool)

```

Comment:
Follow the coding style of the file: add a space after the `if` statement.

Reply:
Just found out - there's also a [codeformatter](https://github.com/dotnet/codeformatter) for C#!